webrender/src/resource_cache.rs

```diff
@@ -117,24 +129,43 @@ impl<K,V> ResourceClassCache<K,V> where K: Clone + Hash + Eq + Debug, V: Resourc
     }
 }
 
+struct ImageResourceRequest {
+    key: ImageKey,
+    rendering: ImageRendering,
+}
+
+struct TextRunResourceRequest {
+    key: FontKey,
+    size: Au,
+    blur_radius: Au,
+    glyph_indices: Vec<u32>,
+}
+
+enum ResourceRequest {
+    Image(ImageResourceRequest),
```

Comment:
I think it would be clearer to just have `Image(ImageKey, ImageRendeing)`, since `ImageResourceRequest` is not used in any other context anyway